SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -14,6 +14,15 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
 
     override fun onViewTaken(view: ApplicationContract.View) {
         this.view = view
+        val stationList = listOf("KGX", "WNS", "WKM", "GLD", "WOK")
         view.setLabel(createApplicationScreenMessage())
+        view.setDepartureDropdown(stationList)
+        view.setArrivalDropdown(stationList)
+    }
+
+    override fun onButtonTapped(departureStation: String, arrivalStation: String, view: ApplicationContract.View) {
+        this.view = view
```

Comment:
You shouldn't need to pass in and set the view here, `this.view` is a variable on the class which you set in `onViewTaken` so should be able to access from any other function on the presenter.